packages/svelte/tests/server-side-rendering/samples/async-context-throws-after-await/_config.js

```diff
@@ -1,6 +1,7 @@
 import { test } from '../../test';
 
 export default test({
+	skip: true, // TODO it appears there might be an actual bug here; the promise isn't ever actually awaited in spite of being awaited in the component
```

Comment:
hmm yeah, it correctly fails in CSR but not in SSR. Will open an issue for this

Reply:
Yeah when I merge this PR I'm going to immediately open two placeholder PRs to immediately fix these